runtime-sdk/src/module.rs

```diff
@@ -304,6 +304,13 @@ pub trait AuthHandler {
         // Default implementation doesn't do anything.
         Ok(())
     }
+
+    /// Perform any action after call, within the transaction context.
+    /// TODO: maybe move into a separate trait, or rename AuthHandler to something like "CallHandler".
```

Comment:
Yeah this should probably be call handler or transaction handler.

Reply:
Done renamed to `TransactionHandler`